examples/DCABot.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+# This file is part of krakenex.
+# Licensed under the Simplified BSD license. See `examples/LICENSE.txt`.
+
+# Made by Jean-Baptiste Pleynet
+# Dollar cost averaging strategy : Use this to regularly buy or sell, in buy & hold objective
+
+
+
+import krakenex
+import sys
+
+texte_key_file = "KEY FILE:"
+texte_pair = "PAIR:"
+texte_type = "TYPE:"
+texte_volume = "VOLUME:"
+succed_string = "result"
+error_string_title = "\n**** ERROR ****"
+
+#print('Argument List:' + str(sys.argv))
+assert len(sys.argv) == 2
+
+fichier_config = sys.argv[1]
+
+file = open(fichier_config, "r") 
+data = file.readlines() 
+
+trouve_key_file = False
+trouve_pair = False
+trouve_type = False
+trouve_volume = False
+
+chaine_erreur = ""
+
+for ligne in data:
```

Comment:
This is a rather convoluted way of reading in configuration. :)

If there's a file with python syntax, e.g. `DCABot.config.py`, it can be `import`ed and used directly. E.g., like [this](http://stackoverflow.com/a/924866/7056908).